test/browser/clubs-page.test.jsx

```diff
@@ -283,7 +309,7 @@ describe("ClubsPage.ModalAddOrChangeYourClub", function() {
           validationErrors: ['old error that should be removed'],
           name: 'blorpy',
           location: 'chicago',
-          website: 'http://example.org',
+          website: 'example.org',
```

Comment:
This has the side effect of ensuring that submitting the form normalizes the `website` field into a proper URL.